### PR TITLE
Release/v0.9.1.49

### PR DESCRIPTION
**Pull request by davidpofo**

### CODE REVIEW

File: controls/views.py [hunk 1]
```diff
@@ -832,6 +840,11 @@ def restore_to_history(request, smt_id, history_id):
         historical_smt = smt.history.get(history_id=history_id)
         # saving historical statement as a new instance
         historical_smt.instance.save()
+        # Update the reason for the new statement record
+        update_change_reason(smt.history.first().instance, change_reason)
+
+        logger.info( f"Change reason: {change_reason}")
```
Comment (gregelin) [inline controls/views.py:846]: There's a pattern to logger messages that generates a log entry in JSON:

- event (an identifiable label/name for the event)
- object (on which event occurred)
- other info (optional)
- user (who made the change)

Example in `controls.views.py` L2030
```
def assign_baseline(request, system_id, catalog_key, baseline_name):
    """Assign a baseline to a system root element thereby showing selected controls for the system."""

    system = System.objects.get(pk=system_id)
    #system.root_element.assign_baseline_controls(user, 'NIST_SP-800-53_rev4', 'low')
    assign_results = system.root_element.assign_baseline_controls(request.user, catalog_key, baseline_name)
    if assign_results:
        messages.add_message(request, messages.INFO,
                             'Baseline "{} {}" assigned.'.format(catalog_key.replace("_", " "), baseline_name.title()))
        # Log start app / new project
        logger.info(
            event="assign_baseline",
            object={"object": "system", "id": system.root_element.id, "title": system.root_element.name},
            baseline={"catalog_key": catalog_key, "baseline_name": baseline_name},
            user={"id": request.user.id, "username": request.user.username}
        )
    else:
        messages.add_message(request, messages.ERROR,
                             'Baseline "{} {}" assignment failed.'.format(catalog_key.replace("_", " "),
                                                                          baseline_name.title()))

    return HttpResponseRedirect("/systems/{}/controls/selected".format(system_id))

```

**Comment by davidpofo** [comment]
The siteapp tests have an erroneous failure in the [build ](https://app.circleci.com/pipelines/github/GovReady/govready-q/1663/workflows/d8eaa76d-7c17-44de-a39b-d2354cd4b879/jobs/3807) that will be looked at with the rest of this cicd - django test related issues.

File: controls/views.py (hunk 1 above)
Comment (davidpofo) [reply controls/views.py:846]: Good idea will address in the next iteration of statement history

Comment (gregelin) [reply controls/views.py:846]: Great

**Review by gregelin — APPROVED** [review]
Looks good. I concur and saw that the tests were previously passing.

**Pull request merged**